e2e/browser/helpers/authorizeClientApp.ts

```diff
@@ -15,9 +15,6 @@ export async function authorizeNss() {
 }
 
 export async function authorizeEss() {
-  const approveButton = Selector("button").withText("Approve");
+  const approveButton = Selector('button[form="approve"]');
```

Comment:
Made just a quick comment to the effect that the 'better' selection isn't working (but we don't know why yet)...?

Reply:
Fair enough! 750b6070